modules/platform/rest-api/nuxeo-rest-api-server/src/main/java/org/nuxeo/ecm/restapi/server/jaxrs/BatchUploadObject.java

```diff
@@ -232,7 +232,7 @@ protected Response uploadNoTransaction(@Context HttpServletRequest request,
         // TODO NXP-18247: should be set to the actual number of bytes uploaded instead of relying on the Content-Length
         // header which is not necessarily set
         long uploadedSize = getUploadedSize(request);
-        boolean isMultipart = contentType != null && contentType.contains("multipart");
+        boolean isMultipart = contentType != null && contentType.contains("multipart/form-data");
```

Comment:
is this required for the fix?

Reply:
Yes. This is half the fix actually.
Otherwise multipart/related will give true and the `FormData#getMultiPartItems` will be called and fail

Reply:
We changed the approach. Now multipart is completely unsupported on master. 
It still fails on 10 and 9 but is disablable